packages/lesmis-server/src/routes/UsersRoute.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { Request, Response, NextFunction } from 'express';
+import uniqby from 'lodash.uniqby';
+import { Route } from '@tupaia/server-boilerplate';
+import { MeditrakConnection } from '../connections';
+
+const FIELDS: Record<string, string> = {
+  user_id: 'id',
+  'user.creation_date': 'createdAt',
+  'user.first_name': 'firstName',
+  'user.last_name': 'lastName',
+  'user.email': 'email',
+  'user.mobile_number': 'mobileNumber',
+  'user.employer': 'employer',
+  'user.position': 'position',
+  'entity.name': 'entityName',
+  'entity.code': 'entityCode',
+  'permission_group.name': 'permissionGroupName',
+  id: 'userEntityPermissionId',
+};
+
+const PERMISSION_GROUPS = ['Laos Schools Admin', 'LESMIS Public', 'Laos Schools Super User'];
+
+// Remove duplicat records and convert user keys to nice camelCase keys for the front end
+const getProcessedUserData = (userData: { user_id: number }[]) =>
+  uniqby(userData, 'user_id').map((user: Record<string, string[]>) => {
+    const newUser: Record<string, string[]> = {};
+    const keys = Object.keys(user);
+    keys.forEach(key => {
+      const newKey = FIELDS[key];
+      newUser[newKey] = user[key];
+    });
+    return newUser;
+  });
+
+export class UsersRoute extends Route {
+  private readonly meditrakConnection: MeditrakConnection;
+
+  constructor(req: Request, res: Response, next: NextFunction) {
+    super(req, res, next);
+
+    this.meditrakConnection = new MeditrakConnection(req.session);
+  }
+
+  async buildResponse() {
+    const userData = await this.meditrakConnection.getUserEntityPermissions({
+      pageSize: '10000',
+      columns: JSON.stringify(Object.keys(FIELDS)),
+      sort: JSON.stringify(['user.creation_date DESC']),
+      filter: JSON.stringify({
+        'entity.name': { comparator: 'like', comparisonValue: 'Laos', castAs: 'text' },
```

Comment:
```suggestion
        'entity.name': 'Laos',
```
I think that should also work? The `castAs` in particular shouldn't be necessary, I believe that was a fix for enums

Also it's probably safer to filter on the code, in case they change the name to e.g. 'Lao People's Democratic Republic'

```suggestion
        'entity.code': 'LA',
```